src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/ApiShould.cs

```diff
@@ -2242,6 +2242,70 @@ public void TestCommandCustomKeyStoreProviderDuringAeQuery(string connectionStri
             }
         }
 
+        [ConditionalTheory(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringSetupForAE))]
+        [ClassData(typeof(AEConnectionStringProvider))]
+        public void TestRetryWhenAEParameterMetadataCacheIsStale(string connectionString)
+        {
+            if (DataTestUtility.EnclaveEnabled)
```

Comment:
if EnclaveEnabled check should be done above in Conditional theory, not in test. We don't want tests to pass if enclave was not enabled, they should not run.